localisation/00_zuwc_united_localisation.csv

```diff
@@ -3132,18 +3132,23 @@ EVTOPTA_WCPAN_200_mogu;Pandaria is my birthright! Rise mogu!;;;;;;;;;;;;;x
 EVTOPTB_WCPAN_200_mogu;We are the storm! Rise mogu!;;;;;;;;;;;;;x
 EVTDESC_WCPAN_200_generic;Ten thousand years ago, the pandaren empire would dissolve, with power shifting to the local level. Tribes and villages that previously answered to the empire were again free to choose their own course. Somewhat inevitably, the races of Pandaria drew inward.\nBut despite that, the people of Pandaria have not forgotten their roots. Many of the races continue to honour figures like Shaohao, Rassharom, and the Monkey King, who ushered in eras of harmony and unprecedented peace. Even the xenophobic yaungol and saurok remember the glory and wealth of the old empire, and seek to surpass it.\nToday, those dreams have become reality, as [target_emperor.GetTitledFirstName] has declared the formation of a new empire. The days of imperial unity have returned to Pandaria.;;;;;;;;;;;;;x
 EVTOPTA_WCPAN_200_generic;Pandaria stands united!;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
-;;;;;;;;;;;;;;x
+EVTTITLE_WCDAC_500;The Heart of the Vale;;;;;;;;;;;;;x
+EVTDESC_WCSHA_500;Some archaeologists ask for your audience. They say that opening excavation sites in the Vale of Eternal Blossoms may be a good idea. The Vale is full of titanic complexes and vaults.\n\nLegends tell that one of the vaults stores the piece of a dead Old God, Y'Shaarj. More precisely, his giant heart has been buried there after Aman'Thul heaved the Old God from the surface of the world.\n\nObtaining such an artefact will help to understand the origin and abilities of the Old Gods.;;;;;;;;;;;;;x
+EVTOPTA_WCSHA_500;Fine, I'll sponsor them.;;;;;;;;;;;;;x
+EVTOPTB_WCSHA_500;Go away.;;;;;;;;;;;;;x
+EVTDESC_WCSHA_510;Finally, you have received news from the excavation sites in the Vale of Eternal Blossoms.\n\nArchaeologists tell that they unearthed a massive building with enormous doors and security systems. It looks like the titans didn't want this vault to be ever opened. A couple of prospectors died from traps or went missing. People are fearful.\n\nWe need to spend more gold to attract workers and purchase equipment. Otherwise, the excavation has to stop.;;;;;;;;;;;;;x
+EVTOPTA_WCSHA_510;Fine, I'll invest more.;;;;;;;;;;;;;x
+EVTOPTB_WCSHA_510;Call prospectors back.;;;;;;;;;;;;;x
```

Comment:
```suggestion
EVTOPTB_WCSHA_510;Call the prospectors back.;;;;;;;;;;;;;x
```

Reply:
Fixed in ce9d0e0c8aaa677cbebc34f9c3a6bf9185bb464e.